src/environment.config.js

```diff
@@ -0,0 +1,24 @@
+const DEV_ENVIRONMENT = {
+  apiBaseUrl: 'http://localhost:8080/'
+}
+
+const PROD_ENVIRONMENT = {
+  apiBaseUrl: 'http://api.wohnen.integreat-app.de/v0/'
+}
+
+const browserEnvironment = localStorage.getItem('environment')
+
+let environment
+
+if (browserEnvironment) {
+  environment = browserEnvironment
+  // eslint-disable-next-line
```

Comment:
why do you have to disable eslint here?

Reply:
Because __DEV__ is a variable replaced by webpack